backend/api/tests/test_models.py

```
@@ -0,0 +1,222 @@
+    def test_viewer_profile_str(self):
+        # Test the string representation
+        user = User.objects.create_user(
+            email="viewer1@example.com",
+            firebase_uid="vieweruid1"
+        )
+        profile = ViewerProfile.objects.create(user=user)
+        assert str(profile) == f"Profile for viewer1@example.com"
+
+@pytest.mark.django_db
+class TestVideoModel:
+    def test_video_creation(self):
```

Ruff (0.8.2)

Missing return type annotation for public function `test_video_creation`

Add return type annotation: `None`

(ANN201)